demos/demo_references.bib

```diff
@@ -25,6 +25,14 @@ @InBook{2015:lmscup
                   {http://www1.maths.leeds.ac.uk/~matak/lms-cup2015.pdf}
 }
 
+@Manual{Gidel:2017,
```

Comment:
Update: The paper has been published in Volume 24, pages 43-60 (doi:10.5194/npg-24-43-2017). Can you make the change?